The CI failed but it was **NOT** related to this PR.

```cpp
test_linalg_cholesky_inverse .....***Failed

C++ Traceback (most recent call last):
--------------------------------------
paddle::pybind::eager_api_inverse(_object*, _object*, _object*)
inverse_ad_func(paddle::Tensor const&)
paddle::experimental::inverse(paddle::Tensor const&)
phi::KernelImpl<void (*)(phi::GPUContext const&, phi::DenseTensor const&, phi::DenseTensor*), &(void phi::InverseKernel<double, phi::GPUContext>(phi::GPUContext const&, phi::DenseTensor const&, phi::DenseTensor*))>::VariadicCompute(phi::DeviceContext const&, phi::DenseTensor const&, phi::DenseTensor*)
phi::funcs::MatrixInverseFunctor<phi::GPUContext, double>::operator()(phi::GPUContext const&, phi::DenseTensor const&, phi::DenseTensor*)
phi::funcs::MapMatrixInverseFunctor<phi::GPUContext, double>::operator()(phi::GPUContext const&, double const*, double*, int, int)
Eigen::PartialPivLU<Eigen::Matrix<double, -1, -1, 1, -1, -1> >::~PartialPivLU()

FatalError: `Segmentation fault` is detected by the operating system.
```

--- a/paddle/fluid/inference/tensorrt/trt_int8_calibrator.cc
+++ b/paddle/fluid/inference/tensorrt/trt_int8_calibrator.cc
@@ -41,10 +41,10 @@ TRTInt8Calibrator::TRTInt8Calibrator(
     int num_ele = data_size / sizeof(int16_t);
     framework::DDim data_shape = common::make_ddim({num_ele});
     temp_tensor.Resize(data_shape);
-    data_tensors_.push_back(temp_tensor);
     data_buffers_[input_name] = std::pair<void*, size_t>(
         static_cast<void*>(temp_tensor.mutable_data<int16_t>(place)),
         data_size);
+    data_tensors_.push_back(temp_tensor);
   }
 }
 

